src/Serilog.Sinks.RabbitMQ/LoggerConfigurationRabbitMQExtension.cs

```diff
@@ -25,9 +25,9 @@
 namespace Serilog;
 
 /// <summary>
-/// Extension method to configure Serilog with a Sink for RabbitMq.
+/// Extension method to configure Serilog with a Sink for RabbitMQ.
 /// </summary>
-public static class LoggerConfigurationRabbitMqExtension
+public static class LoggerConfigurationRabbitMQExtension
```

Comment:
This is potentially a breaking change. Should this be mentioned in the Breaking changes section of the Readme?

Reply:
done